cogs/menudocs.py

```diff
@@ -185,6 +187,23 @@ async def pypi(self, ctx):
         )
         await ctx.send(embed=embed)
 
+    @commands.command(name='format')
+    @ensure_is_menudocs_guild()
+    async def _format(self, ctx):
+        """Sends a helpful embed about how to correctly format your question."""
+        embed = nextcord.Embed(
+            title="Uh oh, could you do us a favour please and re-phrase your question?",
+            description="""
+                > Looks like we are missing some information to properly help you out!\n
+                __Please make sure you have the following:__\n
+                • The code related to your question.
+                • Any errors that you are receiving.
+                • The taceback of your error. (If relevant)
+                • The expected action/output and recieved action/output.
+                """,
+        )
+        await ctx.send(embed=embed)
```

Comment:
Care to add a check which checks if ctx.message is a reply to someone else, then Pyro can also reply to that message. In a sense pinging them?